frontend/src/features/projects/assess/components/ReviewApproveActions.tsx

```diff
@@ -56,6 +56,9 @@ export const ReviewApproveActions = ({
             validateForm().then((errors: any) => {
               if (Object.keys(errors).length === 0) {
                 setApproveERP(true);
+              } else {
```

Comment:
This is really dumb, but by default the validateForm function does not touch all the fields and so the errors won't display. This change forces those errors to display if any exist but pretending to try and submit the form even though we know it'll fail validation (which touches all the fields).